src/components/layout/responsive-row.tsx

```diff
@@ -7,17 +7,44 @@ const PanelContainer = styled.div`
   ${({ theme }) =>
     theme.screens.sm
       ? css`
-          flex-direction: column;
+          flex-flow: column;
```

Comment:
unnecessary change...flex-flow without flow property is as good as flex-direction